internal/collector/telemetry/process_telemetry.go

```diff
@@ -0,0 +1,105 @@
+package telemetry
+
+import (
+	"context"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/prometheus/procfs"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// ProcessMetricsViews is a struct that contains views related to process metrics (cpu, mem, etc)
+type ProcessMetricsViews struct {
+	views []*view.View
+	done  chan struct{}
+}
+
+var mRuntimeAllocMem = stats.Int64("oc.io/process/memory_alloc", "Number of bytes currently allocated in use", "By")
```

Comment:
nit: "B" instead of "By"

Reply:
According to the documentation, it should be By (for bytes) http://unitsofmeasure.org/ucum.html#para-48
https://opencensus.io/stats/measure/